app/controllers/api/functions.php

```diff
@@ -417,6 +417,21 @@
             ->setParam('document', $function->getArrayCopy())
         ;
 
+        $tags = $projectDB->getCollection([
```

Comment:
This is a loop we better manage inside the worker as it has the potential to contain unlimited amounts of tags. Use the deletes worker as a reference to view how we delete big lists of data using chunks rather than in one go.